### PR TITLE
follow orbit and update gson to 2.8.9

### LINKED ISSUE #588: Maybe updating to gson 2.8.9?
Changelog:

image

It is in general good to keep track of newer versions of upstream dependencies so downstream users won't have to use multiple versions of the same library

### PR DESCRIPTION
**Pull request by cdietrich**
follow orbit and update gson to 2.8.9
Fixes #588

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
Blocked until Orbit 2022-03 M2 is out

File: releng/releng-target/lsp4j.target.target [hunk 1]
```diff
@@ -3,9 +3,9 @@
 <target name="org.eclipse.xtext.helios.target" sequenceNumber="0">
 	<locations>
 		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/tools/orbit/downloads/2021-09"/>
-			<unit id="com.google.gson" version="2.8.7.v20210624-1215"/>
-			<unit id="com.google.gson.source" version="2.8.7.v20210624-1215"/>
+			<repository location="https://download.eclipse.org/tools/orbit/downloads/2022-03"/>
```
Comment (jonahgraham) [inline releng/releng-target/lsp4j.target.target:6]: Use `https://download.eclipse.org/tools/orbit/downloads/latest-I/` for now and we can change it later (add it to the checklist on https://github.com/eclipse/lsp4j/issues/541 so it isn't forgotten)

**Review by jonahgraham — APPROVED** [review]
LGTM once it builds.

File: releng/releng-target/lsp4j.target.target (hunk 1 above)
Comment (cdietrich) [reply releng/releng-target/lsp4j.target.target:6]: i would simply wait the 2 weeks for m2

**Pull request merged**